pkg/cfg/resolver/gotemplate.go

```diff
@@ -0,0 +1,68 @@
+package resolver
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"html/template"
```

Comment:
Don't we want to use `text/template` instead?
https://golang.org/pkg/text/template/

Reply:
good point